spec/features/idv/analytics_spec.rb

```diff
@@ -70,7 +70,8 @@
       can_pass_with_additional_verification: false,
       attributes_requiring_additional_verification: [],
       vendor_name: 'ResolutionMock',
-      vendor_workflow: nil }
+      vendor_workflow: nil,
+      verified_attributes: nil }
```

Comment:
💭 

```suggestion
      verified_attributes: nil,
    }
```

I has a similar situation in my current PR, where I wished there had been a trailing comma. I debated reformatting the block to allow me to include one, but decided against it. Seeing you run into the same thing makes me wonder if maybe the idea has some merit?

(Unimportant nit.)

Reply:
I think maybe in a separate PR would be good